src/external_app/external_messaging.ts

```diff
@@ -304,6 +312,7 @@ export interface ExternalConfig {
   hasBarCodeScanner: number;
   canSetupImprov: boolean;
   downloadFileSupported: boolean;
+  addEntityToSupported: boolean;
```

Comment:
We are in a weird situation where `downloadFileSupported` differs from the rest of params, but I think we could use this to match the rest:

```suggestion
  canAddEntityToApp: boolean;
```